docs/gettingstarted/index.rst

```diff
@@ -105,10 +105,10 @@ Getting started with Flocker
 
          .. version-code-block:: console
 
-            you@laptop:~$ sudo yum install -y @buildsys-build python python-devel python-virtualenv && \
-              virtualenv flocker-tutorial && \
-              flocker-tutorial/bin/pip install --upgrade pip && \
-              flocker-tutorial/bin/pip install --quiet https://clusterhq-archive.s3.amazonaws.com/python/Flocker-|latest-installable|-py2-none-any.whl && source flocker-tutorial/bin/activate
+            you@laptop:~$ sudo yum install -y @buildsys-build python python-devel python-virtualenv libffi-devel openssl-devel && \
+              curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/linux-install.sh && \
+              sh linux-install.sh && \
```

Comment:
As written this will still run `sh` on some errors spat out by an HTTP server.  The essential point isn't the use of `|` it's the blind use of a shell on some bytes dumped into a file from the internet...

Although now I realize **this** section of the docs still uses `&&`.  Too bad we're inconsistent about that.  Anyway, not a blocking issue.